src/index.ts

```diff
@@ -0,0 +1,7 @@
+import * as cma from './cma';
```

Comment:
This entire file can be replaced with `export * as cma from './cma';`

Reply:
I'm not sure that's what we want - isn't the intent of what we're talking about doing with index to *explicitly* export methods in our modules?

Reply:
In that case:

```typescript
export {
  generateCMASpawnArguments,
  invokeCumulusMessageAdapter,
  runCumulusTask
} from './cma';
```

Reply:
In that case, 👍 : e89700f